python/lsst/qserv/tests/qservDbLoader.py

```diff
@@ -120,8 +120,9 @@ def prepareDatabase(self):
         self.czar_wmgr.createDb(self._dbName)
 
         if self.multi_node:
-            self.wmgr.dropDb(self._dbName, mustExist=False)
-            self.wmgr.createDb(self._dbName)
+            for node in self.nodeNames:
+                self.nWmgrs[node].dropDb(self._dbName, mustExist=False)
+                self.nWmgrs[node].createDb(self._dbName)
```

Comment:
This would become:

    for wmgr in self.nWmgrs.values():
        wmgr.dropDb(self._dbName, mustExist=False)
        wmgr.createDb(self._dbName)